src/components/TeamMember/index.tsx

```diff
@@ -5,24 +5,50 @@ import { StaticImage } from 'gatsby-plugin-image'
 import React from 'react'
```

Comment:
@corywatilo Heads up that I've made changes to the team member component to optionally only show the first name ([issue](https://github.com/PostHog/posthog.com/issues/9727))

I tested that the component displays the same without this parameter
<img width="475" alt="Screenshot 2024-10-31 at 2 59 09 PM" src="https://github.com/user-attachments/assets/e184eea8-8fb1-4274-97df-100b854c7111">

with onlyShowFirstName = true:
<img width="733" alt="Screenshot 2024-10-31 at 3 00 45 PM" src="https://github.com/user-attachments/assets/04139a4f-bea2-43cf-957f-9ce6666cfe68">